UKHomeOffice/nmsw-ui
# NMSW-1061 Add Details ETA section to UPT results page

# Ticket



----

## AC

ETA details section for UPT check results page

----

## To test

- go to UPT status check results page
- > there should be a details link
- click the link
- > ETA help info should be shown
- click the link again
- > ETA help info should be hidden

----

## Notes
